addons/binding/org.openhab.binding.feed/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="feed" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- DEFINITIONS of terms used in the binding: 
+	Feed is a XML document used for providing users with frequently updated content. 
+		The most popular feed formats are RSS and Atom. 
+	Entry is a single element in the Feed, that contains reference (link), short 
+		description and other information like images, comments and etc. Entry in this binding is abstraction for RSS item element 
+		and Atom entry element. -->
+
+	<!-- Feed Thing Type -->
+	<thing-type id="feed">
+		<label>Feed</label>
+		<description>Provides information about a feed.</description>
+
+		<channels>
+			<channel id="latest-entry" typeId="latest-entry" />
+			<channel id="author" typeId="author" />
+			<channel id="description" typeId="description" />
+			<channel id="title" typeId="title" />
+			<channel id="last-update" typeId="last-update" />
+			<channel id="number-of-entries" typeId="number-of-entries" />
+		</channels>
+
+		<config-description>
+			<parameter name="URL" type="text" required="true">
+				<label>Feed URL</label>
+				<description>The URL of the feed</description>
+			</parameter>
+
+			<!--After the refresh time interval expires, the bindings checks for updates in the Feed, and if the information is not 
+				up to date, updates the feed content stored in the channel -->
+
+			<parameter name="refresh" type="integer">
+				<label>Refresh time interval</label>
+				<description>Refresh time interval in minutes.</description>
+				<default>20</default>
+			</parameter>
+
+		</config-description>
+	</thing-type>
+
+	<channel-type id="latest-entry">
+		<item-type>String</item-type>
+		<label>Latest entry</label>
+		<description>Contains title, description and published date for the last feed entry.</description>
+	</channel-type>
+
+	<channel-type id="author">
+		<item-type>String</item-type>
+		<label>Author</label>
+		<description>The name of the feed author, if author is present</description>
+	</channel-type>
+
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>The title of the feed</description>
+	</channel-type>
+
+	<channel-type id="description">
+		<item-type>String</item-type>
+		<label>Description</label>
+		<description>Description of the feed</description>
+	</channel-type>
+
+	<channel-type id="last-update">
```

Comment:
Ok, I see that feeds provide this information, so ok for me to keep it, but it should imho also be marked advanced.